src/odemis/gui/util/img.py

```diff
@@ -886,6 +886,7 @@ def calculate_ticks(value_range, client_size, orientation, tick_spacing):
         return tick_list, vtp_ratio
 
     vtp_ratio = abs(pixel_space / value_space)
+    epsilon = vtp_ratio / 10
```

Comment:
you divide by 10 because ...? Maybe add a comment for the next reader of the code :)
if value to pixel ratio smaller than epsilon it must be the zero?